samples/angular/src/app/app.component.ts

```diff
@@ -0,0 +1,90 @@
+import { Component, ViewChild, ElementRef, OnInit } from '@angular/core';
+import { Validators, FormBuilder } from '@angular/forms';
+import { data } from '../mock-data';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.less']
+})
+export class AppComponent implements OnInit {
+  public loading = true;
+  public profileForm = this.fb.group({
+    name: ['', Validators.required],
+    gender: [''],
+    birthdate: [''],
+    address: [''],
+    email: ['', [Validators.required, Validators.email]],
+    job: [''],
+    mailLetter: [true]
+  });
+  public chartType = 'line';
+  public checked = false;
+  
+  public onTabChangeHandler = this.onTabChange.bind(this);
+
+  @ViewChild('chart') chart!: ElementRef;
+  @ViewChild('theme') theme!: ElementRef;
+  @ViewChild('profile') profileBtn!: ElementRef;
+  @ViewChild('dlg') dlg!: ElementRef;
+  constructor (private fb: FormBuilder) {}
+
+  async ngOnInit (): Promise<void> {
+    const defaultTheme = sessionStorage.getItem('elf-theme') || 'light';
+    await this.loadTheme(defaultTheme);
+    this.loading = false;
+    this.checked = defaultTheme === 'dark';
+  }
+
+  ngAfterViewInit (): void {
+    this.generateChart();
+  }
+  
+  async loadTheme (theme: string): Promise<unknown> {
+    document.documentElement.setAttribute('theme', theme);
+    document.documentElement.style.setProperty('--theme', theme);
```

Comment:
Do we need this line?

Reply:
We can remove these lines.

Reply:
Actually, we can remove only line 45. Line 44 is require for theme switching.